### PR TITLE
Ensure repo files using yum_repository

### PR DESCRIPTION
**Pull request by m-bull**
Don't install the OpenHPC release rpm to get yum repo files, make them with `yum_repository` instead. 

This allows the specification of a custom ohpc package repo URL, such as one pointing to an internal package mirror.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -48,9 +49,52 @@ openhpc_enable:
...
+      gpgkey: "https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7"
+  "8":
+    - name: epel
+      file: epel.repo
+      description: "Extra Packages for Enterprise Linux 8 - $basearch"
+      metalink: "https://mirrors.fedoraproject.org/metalink?repo=epel-8&arch=$basearch&infra=$infra&content=$contentdir"
+      gpgcheck: true
+      gpgkey: "https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-8"
+
+extra_ohpc_extra_repos: []
```
Comment (sjpb) [inline defaults/main.yml:96]: That name seems a bit odd

File: defaults/main.yml [hunk 2]
```diff
@@ -48,9 +49,52 @@ openhpc_enable:
...
+      gpgcheck: true
+      gpgkey: https://raw.githubusercontent.com/openhpc/ohpc/v2.6.1.GA/components/admin/ohpc-release/SOURCES/RPM-GPG-KEY-OpenHPC-2
+    - name: OpenHPC-updates
+      file: OpenHPC.repo
+      description: OpenHPC-2 - Updates
+      baseurl: "http://repos.openhpc.community/OpenHPC/2/updates/CentOS_8"
+      gpgcheck: true
+      gpgkey: https://raw.githubusercontent.com/openhpc/ohpc/v2.6.1.GA/components/admin/ohpc-release/SOURCES/RPM-GPG-KEY-OpenHPC-2
+
+default_ohpc_extra_repos:
```
Comment (sjpb) [inline defaults/main.yml:80]: should start `ohpc_`?

File: defaults/main.yml (hunk 1 above)
Comment (sjpb) [reply defaults/main.yml:96]: How about having

- `ohcp_repos` as the FINAL list of repos, made up of concatenating:

  - `ohpc_openhpc_repos` - (currently `ohpc_repos`)
  - `ohpc_default_extra_repos` - i.e. epel, (currently `default_ohpc_extra_repos`)
  - `ohcp_extra_repos` - defaulting to `[]`

**Review by sjpb — CHANGES_REQUESTED** [review]

File: defaults/main.yml (hunk 1 above)
Comment (m-bull) [reply defaults/main.yml]: Yeah, good idea - have implemented this but made `ohpc_extra_repos` into `openhpc_extra_repos` as an exposed, configurable var.

File: defaults/main.yml (hunk 2 above)
Comment (m-bull) [reply defaults/main.yml]: Updated, thanks.

**Review by sjpb — APPROVED** [review]

**Pull request merged**